pages/edit.js

```diff
@@ -0,0 +1,365 @@
+          >
+            <Image alt="remove" src={remove} width="64" height="64" />
+          </DeleteButton>
+        </ButtonWrapper>
+        <ApplyWrapper>
+          <DummyDiv />
+          <Apply type="submit">
+            <Image alt="hook" src={hook} width="42" height={41.71} />
+            <div>Apply</div>
+          </Apply>
+        </ApplyWrapper>
+      </form>
+      <DownGap />
+    </>
+  );
+}
+
+const HeadlineWrapper = styled.div`
+  position: fixed;
+  width: 100%;
+  backdrop-filter: blur(10px);
+`;
+
+const TopGap = styled.div`
+  height: 75px;
+`;
+
+const FormList = styled.ol`
+  background-color: rgba(52, 108, 61, 0.7);
+  margin: 5px 10vw;
+  border-radius: 5px;
+  padding: 15px 0;
+  list-style-type: none;
+  counter-reset: li;
+`;
+
+const Li = styled.li`
+  display: flex;
+  justify-content: center;
+
+  &::before {
+    counter-increment: li;
+    content: counter(li, decimal-leading-zero);
+    color: #caf6ff;
+    margin: 0 5px 0 0;
+  }
+`;
+
+const InputAndLabelWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+const ExerciseName = styled.input`
+  background-color: #edffdf;
+  margin: 0 1px;
+  border-radius: 5px;
+  border: none;
+  width: 30vw;
+  max-width: 340px;
+`;
+
+const Select = styled.select`
+  text-align: right;
+  background-color: #edffdf;
+  width: 50px;
+  margin: 0 1px;
+  border-radius: 5px;
+  border: none;
+`;
+
+const Label = styled.label`
+  color: #caf6ff;
+  font-weight: 300;
+  font-size: 0.8rem;
+`;
+
+const ButtonWrapper = styled.div`
+  display: flex;
+  justify-content: center;
+`;
+
+const AddButton = styled.button`
+  background-color: transparent;
+  border-radius: 60px;
+  border: none;
+
+  /* &:hover {
+    background: radial-gradient(
+      circle,
+      rgba(85, 255, 5, 0.1),
+      rgba(85, 255, 5, 0.8) 55%,
+      transparent 72%
+    );
+  } */
+  &:active {
+    transform: translateY(1px);
+    background: radial-gradient(
+      circle,
+      rgba(85, 255, 5, 0.1),
+      rgba(85, 255, 5, 0.8) 55%,
+      transparent 72%
+    );
+  }
+`;
+
+const DeleteButton = styled.button`
+  background-color: transparent;
+  border-radius: 60px;
+  border: none;
+
+  /* &:hover {
+    background: radial-gradient(
+      circle,
+      rgba(255, 58, 0, 0.1),
+      rgba(255, 58, 0, 0.8) 55%,
+      transparent 72%
+    );
+  } */
+  &:active {
+    transform: translateY(1px);
+    background: radial-gradient(
+      circle,
+      rgba(255, 58, 0, 0.1),
+      rgba(255, 58, 0, 0.8) 55%,
+      transparent 72%
+    );
+  }
+`;
+
+const ApplyWrapper = styled.div`
+  display: flex;
+  justify-content: space-around;
+`;
+
+const DummyDiv = styled.div`
+  width: 35vw;
+  max-width: 167px;
+`;
+
+const Apply = styled.button`
+  color: #caf6ff;
+  font-size: 2rem;
+  font-weight: 700;
+  text-align: center;
+  background-color: rgba(52, 108, 61, 1);
+  width: 167px;
+  height: 64px;
+  border: none;
+  border-radius: 5px;
```

Comment:
Again border: none but a border radius. Does that work?

Reply:
It's a styled button also